components/Utilities/NetworkChecker/NetworkChecker.c

```diff
@@ -0,0 +1,67 @@
+#include "NetworkChecker.h"
+
+static const char *TAG = "Utility_Wifi";
+
+/**
+ * @brief Checks the current Wi-Fi connection status.
+ * This function retrieves the information of the currently connected Wi-Fi access point.
+ * If connected, it logs the SSID; otherwise, it indicates no connection.
+ * @return bool Returns true if connected to a Wi-Fi network, false otherwise.
+ */
+bool CheckWifiStatus()
```

Comment:
`IsWifiConnected()` would be a better name for this function

Reply:
I will change it.